content/en/docs/test.md

```diff
@@ -57,79 +57,99 @@ culpa qui officia deserunt mollit anim id est laborum.
 
 ## Lists
 
+Markdown doesn't have strict rules about how to process lists. When we moved
+from Jekyll to Hugo, we broke some lists. To fix them, keep the following in
+mind:
+
+- Make sure you indent sub-list items **4 spaces** rather than 2 that you may be
+  used to. Counter-intuitively, you need to indent block-level content within a
+  list item an extra 4 spaces too.
+
+- To end a list and start another, you need a HTML comment block on a new line
+  between the lists, flush with the left-hand border. The first list won't end
+  otherwise, no matter how many blank lines you put between it and the second.
+
 ### Bullet lists
 
 - This is a list item
 * This is another list item in the same list
 - You can mix `-` and `*`
-  - To make a sub-item, indent two spaces.
-    - This is a sub-sub-item.
-
+    - To make a sub-item, indent two tabstops (4 spaces). **This is different
+      from Jekyll / Kramdown.**
```

Comment:
Nit: remove spaces padding the slash.

Reply:
I think it's more readable as it is, or replacing the slash with `and` since Jekyll and Kramdown are not equivalent words. WDYT?